README.md

```diff
@@ -1,6 +1,14 @@
-# CSS Is Fun
-### Yes! You heard it right CSS is awesome, and this repository contains CSS UI Snippets
-CSS-Is-Fun is a collection of creative and fun CSS animations and effects to enhance your web projects.
+# CSS Is Fun 
+
+### Yes! You heard it right CSS is awesome, That's great to hear! CSS is a powerful tool for designing and styling web pages this repository also contains UI snippets are pre-built CSS code blocks that can be easily integrated into a website to add style and functionality.
```

Comment:
Here

... this repository also contains UI snippets `which` are pre-built CSS ...

Ig this sounds better


Just commit these two changes which i mentioned.... rest all looks good ✨

Reply:
@Deveesh-Shetty i have fixed it . check now